nextcord/channel.py

```diff
@@ -794,6 +816,450 @@ def archived_threads(
         :class:`Thread`
             The archived threads.
         """
+
+        return ArchivedThreadIterator(
+            self.id, self.guild, limit=limit, joined=joined, private=private, before=before
+        )
+
+
+class ForumChannel(abc.GuildChannel, Hashable):
+    """Represents a Discord guild forum channel.
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two channels are equal.
+
+        .. describe:: x != y
+
+            Checks if two channels are not equal.
+
+        .. describe:: hash(x)
+
+            Returns the channel's hash.
+
+        .. describe:: str(x)
+
+            Returns the channel's name.
+
+    Attributes
+    ----------
+    id: :class:`int`
+        The snowflake ID of this channel.
+    guild: :class:`Guild`
+        The guild this channel belongs to.
+    name: :class:`str`
+        The name of this channel.
+    category_id: Optional[:class:`int`]
+        The ID of the :class:`CategoryChannel` this channel belongs to, if any.
+    topic: :class:`str`
+        The topic of this channel, if any.
+    position: :class:`int`
+        The position in the channel list, where the first channel is ``0``.
+    nsfw: :class:`bool`
+        If this channel is marked as NSFW.
+    slowmode_delay: :class:`int`
+        The delay in seconds which members must wait between sending messages.
+    flags: :class:`ChannelFlags`
+        The flags that detail features of this channel.
+    default_auto_archive_duration: :class:`int`
+        The archive duration which threads from this channel inherit by default.
+    last_message_id: :class:`int`
+        The snowflake ID of the last message sent to a thread in this channel.
```

Comment:
In the Discord docs I'm seeing:

> The last_message_id field on the forum channel object tracks the id of the most recently created thread. It has the same behavior and requirements as it does for messages, namely that you will not receive a CHANNEL_UPDATE when it is changed. Instead clients should update the value when receiving [Thread Create](https://discord.com/developers/docs/topics/gateway#thread-create).

https://discord.com/developers/docs/topics/threads#forums

Are you sure this is a message id and not a thread id?

Reply:
i believe its the last message of the most recently created thread, either way it would need to be edited